packages/sample-project/src/app/components-services/baseInvoice.service.tsx

```diff
@@ -0,0 +1,97 @@
+import React = require("react");
+import { InvoiceBase, LocalizatonOptions, TableDataProp } from "@leancodepl/invoice-template";
+import { Injectable } from "@nestjs/common";
+
+const localizatonOptions: LocalizatonOptions = {
+    dateFormat: "yyyy-MM-d",
+    documentDate: {
+        label: "Data wystawienia:",
```

Comment:
localization options should be separate from the values. So you can reuse specific localization config. Doing this this way would make it really weird.